tests/acceptance/features/webUIFilesList/fileList.feature

```diff
@@ -1,48 +0,0 @@
-Feature: User can view files inside a folder
-  As a user
-  I want to be able to view folder contents
-  So that I can work with files and folders inside it
-
-  Background:
-    Given user "Alice" has been created with default attributes and without skeleton files in the server
-    And user "Alice" has created folder "simple-folder" in the server
-    And user "Alice" has created folder "simple-empty-folder" in the server
-    And user "Alice" has uploaded file "lorem.txt" to "lorem.txt" in the server
-    And user "Alice" has uploaded file "lorem.txt" to "textfile0.txt" in the server
-    And user "Alice" has uploaded file "data.zip" to "data.zip" in the server
-    And user "Alice" has logged in using the webUI
-
-
-  Scenario: Resources are listed and files list displays quicklink quick action
```

Comment:
Please, add comments if these scenarios have been covered or not. And also provide the link for scenarios where they are covered